solr-updater/src/main/java/dk/dbc/search/solrdocstore/updater/profile/OpenAgencyProfile.java

```diff
@@ -27,28 +30,41 @@
  *
  * @author Morten Bøgeskov (mb@dbc.dk)
  */
-public class Profile implements Serializable {
+public class OpenAgencyProfile implements Serializable {
```

Comment:
Jeg er ikke helt med på hvorfor den her ikke hedder noget ala VipCoreProfile ?

Reply:
Det er backet op af cache, datastrukturer og en databasetabel i solr-doc-store-db. Jeg ville gerne ændre så lidt som muligt. Så for at tabel-navnet er konsistent med datatypen besluttede jeg mig for ikke at ændre det.